examples/torch/common/utils.py

```diff
@@ -166,9 +166,7 @@ def is_staged_quantization(config):
     if isinstance(compression_config, list):
         compression_config = compression_config[0]
     algo_type = compression_config.get("algorithm")
-    if algo_type is not None and algo_type == "quantization" and compression_config.get("params", {}):
-        return True
-    return False
+    return bool(algo_type is not None and algo_type == "quantization" and compression_config.get("params", {}))
```

Comment:
This rule is discussable, in some case it can good in some bad, in some case more readable in other less.
Need to get team's opinion on this rule.

Reply:
New rules that the code violates:
[SIM401](https://docs.astral.sh/ruff/rules/if-else-block-instead-of-dict-get/)
[E721](https://docs.astral.sh/ruff/rules/type-comparison/)
[E266](https://docs.astral.sh/ruff/rules/multiple-leading-hashes-for-block-comment/)
[SIM113](https://docs.astral.sh/ruff/rules/enumerate-for-loop/)
[E262](https://docs.astral.sh/ruff/rules/no-space-after-inline-comment/)
[F841](https://docs.astral.sh/ruff/rules/unused-variable/)
[SIM910](https://docs.astral.sh/ruff/rules/dict-get-with-none-default/)
[E251](https://docs.astral.sh/ruff/rules/unexpected-spaces-around-keyword-parameter-equals/)
[E201](https://docs.astral.sh/ruff/rules/whitespace-after-open-bracket/)
[E231](https://docs.astral.sh/ruff/rules/missing-whitespace/)
[E203](https://docs.astral.sh/ruff/rules/whitespace-before-punctuation/)
[SIM103](https://docs.astral.sh/ruff/rules/needless-bool/)
[E265](https://docs.astral.sh/ruff/rules/no-space-after-block-comment/)

Reply:
Discussed offline. Decided to ignore this rule - [SIM103](https://docs.astral.sh/ruff/rules/needless-bool/)